Sources/DateScrubber.swift

```diff
@@ -140,7 +200,50 @@ public class DateScrubber: UIViewController {
     }
 
     private func setSectionlabelFrame(){
-        self.sectionLabel.frame = CGRectMake(self.view.frame.width - SectionLabel.RightOffsetForSectionLabel - self.sectionLabel.sectionlabelWidth, 0, self.sectionLabel.sectionlabelWidth, viewHeight)
 
+        let rightOffset = self.sectionLabelState == .Active ? SectionLabel.RightOffsetForActiveSectionLabel : SectionLabel.RightOffsetForInactiveSectionLabel
+        self.sectionLabel.frame = CGRectMake(self.view.frame.width - rightOffset - self.sectionLabel.sectionlabelWidth, 0, self.sectionLabel.sectionlabelWidth, viewHeight)
+    }
+
+    private func setScrubberFrame(){
+        let xPos = self.scrubberState == .Active ? self.containingViewFrame.width -  self.scrubberWidth - DateScrubber.RightEdgeInset : self.containingViewFrame.width
```

Comment:
Personally, I'd break this down into more lines of code, easier to follow. Having inline math inside a ternary operator is a bit hard, but that's just me. 😸